config/demo.conf

```diff
@@ -14,7 +14,13 @@
 include "defaults.conf"  // relative path to this file
 
 // write to local storage by default for this demo
+
+
 project_dir = ${JIANT_PROJECT_PREFIX}
+
+word_embs_file =""
```

Comment:
Again, revert all these changes. You should only change lines that are related to this issue.